tools/Build.flow

```diff
@@ -0,0 +1,59 @@
+<title>FOAM Build</title>
+<h1>Pom-O-Matic</h1>
+<p>Solve your Build troubles with Pom-O-Matic!</p>
+
+<h1>Overview</h1>
+
+<p>The FOAM build is a <b>node</b> tool which executes Javascript methods (<b>Tasks</b>) against Project Object Model (<b>POM</b>)(or a more lighthearted acronym expansion: Pom-O-Matic) files.</p>
+
+
+<p><b>POMs</b> are JSON files which identify source code, application configuration, and build instructions. They are hierarchical and can exist at any level of a code repository.</p>
+
+<p>There are two types of POMs:
```

Comment:
Let's expand or rephrase this.

Conceptually there are two types of fields to the pom, tooling fields which set up tasks and environment variables for the build.js process, and build fields which declare build inputs such as files/javaFiles etc.

It would also be useful to add a section about how environment variables work.

From what I gather, environment variables are delcared with default values but can be overriden by environment variables that exist when you run the code?  And environment variables are passed down into tools run by the build process somehow?